zipline/pipeline/factors/technical.py

```diff
@@ -144,6 +148,205 @@ def compute(self, today, assets, out, close, volume):
         out[:] = nanmean(close * volume, axis=0)
 
 
+class CorrelationFactor(SingleInputMixin, CustomFactor):
```

Comment:
I wouldn't include the name `Factor` in either of these.  The user is already importing these from `zipline.pipeline.factors`.